### ISSUE TITLE
Can not change password

### ISSUE THREAD

**Issue by jcisio** [Cyr-ius/hass-livebox-component#32]
Hello,

It seems to be a caching problem. After I changed Livebox password then I updated configuration.yaml, I have an error

```
Logger: custom_components.livebox.bridge
Source: custom_components/livebox/bridge.py:45
Integration: Orange Livebox (documentation, issues)
First occurred: 10:30:43 PM (1 occurrences)
Last logged: 10:30:43 PM

Error Authorization (Starting session failed (APIResponse: {"status": 1, "data": {}})).
```
I change line 45 to log the password, and it shows the old one. I tried to restart HA multiple times, reinstall, uninstall then install the Livebox router integration, but it uses always the old password.

**Comment by jcisio** [comment]
Found it in `.storage/core.config_entries`. I think data from `configuration.yaml` should not be stored there. I'll try to make a PR.

**Comment by jcisio** [comment]
Sorry I haven't found a way to fix it properly. I checked https://github.com/home-assistant/core/blob/dev/homeassistant/components/free_mobile/notify.py as an example, but couldn't figure it out, because it seems that this component could work without change in `configuration.yaml` (but discovery didn't work for me, I had to modify the fiile).

For now I guess I'll just update `.storage/core.config_entries` and call it a day.
